modules/reforgeBidAdapter.js

```diff
@@ -0,0 +1,105 @@
+import * as utils from '../src/utils.js';
+import {config} from '../src/config.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'reforge';
+const ENDPOINT_URL = 'http://use.reforge.in/bid?rtb_seat_id=0011&secret_key=wzP8eKAVkc&type=prebid&appid=40';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER, NATIVE, VIDEO],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function (bid) {
+    return !!(bid.params.placementId);
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @return Array Info describing the request to the server.
+   * @param validBidRequests
+   * @param bidderRequest
+   */
+  buildRequests: function (validBidRequests, bidderRequest) {
+    if (validBidRequests.length === 0) {
+      return [];
+    }
+    return validBidRequests.map(bidRequest => {
+      let sizes = []
+      if (bidRequest.mediaTypes && bidRequest.mediaTypes[BANNER] && bidRequest.mediaTypes[BANNER].sizes) {
+        sizes = bidRequest.mediaTypes[BANNER].sizes;
+      } else if (bidRequest.mediaTypes[VIDEO] && bidRequest.mediaTypes[VIDEO].playerSize) {
+        sizes = bidRequest.mediaTypes[VIDEO].playerSize
+      }
+      const jaySupported = 'atob' in window && 'currentScript' in document;
+      const device = getDevice();
+      const payload = {
+        placementId: bidRequest.params.placementId,
+        sizes: sizes,
+        bidId: bidRequest.bidId,
+        categories: bidRequest.params.categories,
+        referer: bidderRequest.refererInfo.referer,
+        jaySupported: jaySupported,
+        device: device,
+        adapterVersion: 5,
+        mediaTypes: bidRequest.mediaTypes,
+        gdprConsent: bidderRequest.gdprConsent
+      };
+      return {
+        method: 'POST',
+        url: ENDPOINT_URL,
+        data: payload
+      };
+    });
+  },
+
+  /**
+   * Unpack the response from the server into a list of bids.
+   *
+   * @param {ServerResponse} serverResponse A successful response from the server.
+   * @param bidRequest
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function (serverResponse, bidRequest) {
+    const bidResponses = [];
+    const response = serverResponse.body;
+
+    response.forEach(function(bid) {
+      bid.ttl = config.getConfig('_bidderTimeout');
```

Comment:
Are you sure this is what you want, `bidderTimeout` is in milliseconds, while `ttl` is in seconds?